### DISCUSSION TITLE
could not load model

### DISCUSSION THREAD

**Discussion by taotaotao3** [mudler/LocalAI#738] [Q&A]
>Note: The example automatically downloads the gpt4all model as it is under a permissive license. The GPT4All model does not seem to be enough to run AutoGPT. WizardLM-7b-uncensored seems to perform better (with f16: true)

I tried the above sentence point. I wanna use free autogpt. So, I challenged "AutoGPT" + "GPT4ALL".  
Can I use this method? Or Is there something that I can achieve free autogpt?  

My env file is the following
`(transzoom) user@usernoMacBook-Pro autoGPT % cat .env. 
MEMORY_BACKEND=json_file. 
OPENAI_API_KEY=sk---anystringhere. 
OPENAI_API_BASE=http://api:8080/v1. 
Models to preload at start. 
Here we configure gpt4all as gpt-3.5-turbo and bert as embeddings. 
PRELOAD_MODELS=[{"url": "github:go-skynet/model-gallery/gpt4all-j.yaml", "name": "gpt-3.5-turbo"}, { "url": "github:go-skynet/model-gallery/bert-embeddings.yaml", "name": "text-embedding-ada-002"}]`

What I did is the following.
`(transzoom) user@usernoMacBook-Pro autoGPT % docker-compose run --rm auto-gpt`
↓
`I want Auto-GPT to : Research. 
AI Name : Taro. 
Rolo : investigator. 
1: Find the website of the surveyed company “Open AI”. 
2: Search for a page with a company profile on the target website. 
3: Extract the html of the target page. 
4: extract company information from html. 
5: Collect and output the extracted information`

My error is the following.
`openai.error.APIError: could not load model - all backends returned error: 11 errors occurred:  
	* failed loading model. 
	* failed loading model. 
         ....  
 {"error":{"code":500,"message":"could not load model - all backends returned error: 11 errors occurred:\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\n","type":""}} 500 {'error': {'code': 500, 'message': 'could not load model - all backends reåturned error: 11 errors occurred:\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\t* failed loading model\n\n', 'type': ''}} {'Date': 'Mon, 10 Jul 2023 23:33:11 GMT', 'Content-Type': 'application/json', 'Content-Length': '406'}`. 

Originally, it is free, so there is no need to enter OPENAI_API_KEY, right?  
Always thanks so much. If you can please give me advise.

**Comment by galfrylich** [comment]
did you resolve that issue?
